subprojects/plugins/src/main/java/org/gradle/api/plugins/internal/JvmPluginsHelper.java

```diff
@@ -175,8 +177,9 @@ public static void configureDocumentationVariantWithArtifact(
         ConfigurationContainer configurations,
         TaskContainer tasks,
         ObjectFactory objectFactory,
-        FileResolver fileResolver
-    ) {
+        FileResolver fileResolver,
+        TaskDependencyFactory taskDependencyFactory
+        ) {
```

Comment:
❌ Indentation seems off.

Reply:
Fixed in https://github.com/gradle/gradle/pull/22433/commits/9f696b19a4ca50fbcc321eec029ec161c4885ac3